### ISSUE TITLE
 Is this a reference to the Global Platform PIN Unblock Key?

### ISSUE THREAD

**Issue by wszwerc** [usnistgov/FIPS201#404]
### All Fields Are Required
**Organization Name (N/A, if individual)**: DHS

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: Line 1040-1041

**Comment (Include rationale for comment)**: "The operator authenticates the owner of the PIV Card through an independent procedure."
 
**Suggested Change**: What does this mean?  Should this not be a statement consistent with IAL3?  Is this a reference to the Global Platform PIN Unblock Key?
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jricher** [comment]
Duplicate of 218.
